lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -1342,7 +1342,7 @@ p.course-block {
 
 .enter-course-blocked {
   @include box-sizing(border-box);
-  @include float(left);
+  @include float(right);
```

Comment:
why do we need to change existing css?

Reply:
It looked odd that the error message floated was to the left of the gear, and the button that it was replacing was to the right of the settings gear.